tools/applicationinsights-web-snippet/src/snippet.ts

```diff
@@ -0,0 +1,331 @@
+import { AIConfig, AppInsights, Envelope, Fields, ISnippetConfig, ScriptElement } from "./type";
```

Comment:
AIConfig should not exist, it should be importing IConfiguration from applicationinsights-web